keyboards/ergodox_infinity/keymaps/rjhilgefort/keymap.c

```diff
@@ -0,0 +1,372 @@
+ * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
+ * |        | → L0 | → L1 |      |      |      |      |           |      |      |      |      |      |      |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |------|           |------| Home | PGDN | PGUP | End  |      |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |      |           |      |      |      |      |      |      |        |
+ * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
+ *   |      |      |      | TRNS |      |                                       | TRNS |      |      |      |      |
+ *   `----------------------------------'                                       `----------------------------------'
+ *                                        ,-------------.       ,-------------.
+ *                                        |      |      |       |      |      |
+ *                                 ,------|------+------+       |------+------+------.
+ *                                 |      |      |      |       |      |      |      |
+ *                                 |      |      |------|       |------|      |      |
+ *                                 |      |      |      |       |      |      |      |
+ *                                 `--------------------'       `--------------------'
+ */
+[EXA] = LAYOUT_ergodox(
+    // left hand
+    _______, _______,     _______,     _______, _______, _______, _______,
+    _______, DF(BASEMAC), DF(BASELNX), _______, _______, _______, _______,
+    _______, _______,     _______,     _______, _______, _______,
+    _______, _______,     _______,     _______, _______, _______, _______,
+    _______, _______,     _______,     KC_TRNS, _______,
+                                                         _______, _______,
+                                                                  _______,
+                                                _______, _______, _______,
+    // right hand
+    _______,  _______, _______, _______, _______, _______, _______,
+    _______,  _______, _______, _______, _______, _______, _______,
+              KC_HOME, KC_PGDN, KC_PGUP, KC_END,  _______, _______,
+    _______,  _______, _______, _______, _______, _______, _______,
+                       KC_TRNS, _______, _______, _______, _______,
+    _______, _______,
+    _______,
+    _______, _______, _______
+),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    // dynamically generate these.
+    case EPRM:
+      if (record->event.pressed) {
+        eeconfig_init();
+      }
+      return false;
+      break;
```

Comment:
This exists as the `EEP_RST` keycode, already.

Reply:
👍 I wasn't using it- just left over from the "default" layout that I copied from as a base. I removed the references and the event handler in my layout.